src/renderer/features/multisig-wallet-create/flexible-multisig/model/flexible-multisig-create.ts

```diff
@@ -211,15 +209,6 @@ const { $: $multisigFee, $pending: $pendingMultisigFee } = createFeeCalculator({
 
 const $fee = combine($proxyFee, $multisigFee, (proxyFee, multisigFee) => multisigFee.add(proxyFee));
 
-const { $tx, $route } = createComplexTxStore({
-  api: $api,
-  initiator: $initiator,
-  signatory: $signer,
-  accounts: accounts.$list,
-  chain: formModel.$chain,
-  transaction: $coreTx,
-});
-
 // Validation
 const validator = createTxValidator(); //should validate all fee (proxy  deposit + existential)
 const { $errors: $firstErrors } = createTxValidationStore({
```

Comment:
Am I right that `secondErrors` will work on the same input as `firstErrors` do? If so, this is incorrect as those are dependend and the whole validation sequence should be like

withdraw(firstFee)
reserve(create_pure_deposit)
withdraw(secondFee)
withdraw(add_second_proxy_deposit_factor+ed)

I also dont see how we account for  `reserve` and last `withdraw` from the sequence above since `createTxValidator` has empty args

Reply:
Will be fixed in separated PR